build.sh

```diff
@@ -52,11 +52,11 @@ generate_rpm_sources() {
             echo "Generating $TARBALL from $SOURCE_TAG tag"
         fi
 
-        pushd "$SCR_DIR"
+        pushd "$SRC_DIR"
```

Comment:
This was a typo I introduced to patch around RHEL 7 not having support for `-C` with git.